training/src/main/scala/com/airbnb/aerosolve/training/DecisionTreeTrainer.scala

```diff
@@ -72,6 +72,15 @@ object DecisionTreeTrainer {
         .filter(x => x.contains(rankKey))
         .take(candidateSize)
 
+    val ex = examples(0)
+    val numFeatures = LinearRankerUtils.getNumFeatures(ex, rankKey)
```

Comment:
for missing value, the generic pipeline has a missing family, will the current implementation of getNumFeatures function still count the number of element in the missing family and get a full number of the feature or `numFeatures ` will depend on the sample ex sampled?

yay, the model should function as data dependent